css/index.css

```diff
@@ -1,3 +1,294 @@
+    max-width: 900px;
+    width: 100%;
+    margin-left: auto;
+    margin-right: auto;
+}
+nav, header, section {
+    display: flex;
+    padding: 34px 0px 0px;
+}
+nav a, header a, section a {
+    text-decoration: none;
+}
+nav {
+    justify-content: space-between;
+    
+}
+.nav {
+    display: flex;
+    
+}
+nav a {
+    color: #bebebe;
+    padding: 0px 30px;
+    align-self: center;
+    font-weight: bold;
+}
+nav a:first-child {
+    padding-left: 0px;
+}
+
+header a, section a {
+    color: black;
+}
+/*nav a:first-child {
+    margin-left: -33px;
+}
+nav img {
+    margin-left: 88px;
+    margin-top: -10px;
+}*/
+header {
+    border-bottom: 1.5px solid black;
+    padding-bottom: 70px;
+}
+
+header a {
+    border: 1px solid black;
+    margin: 0px 127px 0px;
+    text-align: center;
+    display: flex;
+    font-weight: bold;
+    flex-flow: column;
+    padding: 8px;
+}
+h1 {
+    font-size: 76px;
+    text-align: center;
+    margin-top: 128px;
+    width: 302px;
+    margin-left: 65px;
+    padding: 0px 17px;
+}
+.intro-image {
+    display: flex;
+    flex-direction: row-reverse;
+}
+header img {
+    width: 400px;
+    height: 400px;
+    margin-top: 50px;
+    align-self: flex-end;
+}
+section img {
+    width: 100%;
+    padding-bottom: 30px;
+}
+section a {
+    padding-bottom: 5px;
+    display: inline-block;
+}
+.half {
+    width: 50%;
+}
+#top-section .half:first-child {
+    padding-right: 20px;
+}
+.three {
+    width: 33%;
+    padding: 0px 5px 50px;
+
+}
+#contact {
+    border-top: 1.5px black solid;
+    width: 100%;
+    display: flex;
+    flex-direction: column;
+}
+#contact a, address {
+    padding-bottom: 15px;
+}
+footer {
+    text-align: center;
+    padding-bottom: 30px;
+}
+
+/* Tablet Query */
+
+@media(max-width: 800px) {
+    .wrapper {
+        max-width: 780px;
+    }
+    nav {
+        flex-flow: column-reverse;
+        padding-top: 22px;
+    }
+    .nav {
+        justify-content: space-around;
+    }
+    nav img {
+        align-self: center;
+        padding-bottom: 40px;
+    }
+    .half {
+        width: 100%;
+        display: -webkit-flex;
+        display: -moz-flex;
+        display: -ms-flex;
+        display: -o-flex;
+        display: flex;
+        flex-direction: column;
+    }
+    .intro-image {
+        display: none;
+    }
+    h1 {
+        font-size: 76px;
+        text-align: center;
+        margin-top: 38px;
+        margin-left: 0px;
+        /*padding: 0px 250px;*/
+        align-self: center;
+    }
+    header a {
+        border: 1px solid black;
+        align-self: center;
+        font-weight: bold;
+        flex-flow: column;
+        padding: 8px 43px;
+        letter-spacing: .5pt;
+    }
+    header {
+        margin: 0px 15px;
+    }
+    .section {
+        padding: 0px 15px;
+    }
+    #middle-section img {
+        padding-bottom: 0px;
+    }
+
+}
+
+
+/* Mobile Query */
+
+@media(max-width: 500px) {
+    .wrapper {
+        max-width: 400px;
+    }
```

Comment:
Why are you setting the `max-width` of your `.wrapper` class to `400px`? Your mobile view is closer to the design file without this.